### ISSUE TITLE
Include Project card details into response when listing issue events

### ISSUE THREAD

**Issue by annguyen-qh** [hub4j/github-api#929]
It would be great if the `GHIssueEvent` class could include Project card details. This information is valuable with project-related issue events

The relevant GitHub API endpoint is `GET /repos/:owner/:repo/issues/:issue_number/events` - https://developer.github.com/v3/issues/events/#list-issue-events

**Comment by ki45** [comment]
Hi,
Please correct me - but what is needed here is addition of data member GHProjectCard in GHIssueEvent. I can proceed with this then ?

**Comment by bitwiseman** [comment]
@ki45 
Yes, but you'll need to enable the preview under "Preview notice" in the doc link above. 

Please take this issue and give it a try.  If you get stuck or have questions, drop them here or in the gitter channel.

**Comment by ki45** [comment]
Hi

Apologies for the late reply. I do not understand what is meant by enable preview. This is my first time contributing to a repo someone else owns.
The link above leads to a reference doc for the github rest api. So how do I enable preview ? Please explain once and for posterity.

**Comment by bitwiseman** [comment]
@ki45 
These are preview api:
https://docs.github.com/en/free-pro-team@latest/rest/overview/api-previews

> API previews let you try out new APIs and changes to existing API methods before they become part of the official GitHub API.
>
> During the preview period, we may change some features based on developer feedback. If we do make changes, we'll announce them on the developer blog without advance notice.
>
> To access an API preview, you'll need to provide a custom [media type](https://docs.github.com/en/free-pro-team@latest/rest/overview/media-types) in the Accept header for your requests. Feature documentation for each preview specifies which custom media type to provide.

To see examples of how previews work in this library, look at any method that uses the `withPreview()` method such as: 

https://github.com/hub4j/github-api/blob/a585b4957fbe5e1753ae03b2f9e822ec74b85e99/src/main/java/org/kohsuke/github/GHApp.java#L200-L220

**Comment by ki45** [comment]
Apologies for the late reply. I have enabled preview with the path mentioned by @annguyen-qh  in GHIssueEvent. Please advise on how I can proceed.

![githubapipic](https://user-images.githubusercontent.com/38736421/104847248-ec0dca00-5904-11eb-942a-ac0e2f351223.PNG)

**Comment by bitwiseman** [comment]
@ki45 
Does it work?  
Write some tests!  See the CONTRIBUTING.md file for instructions.
